bzt/modules/locustio.py

```diff
@@ -82,7 +82,7 @@ def startup(self):
         load = self.get_load()
         concurrency = load.concurrency or 1
         if load.ramp_up:
-            hatch = math.ceil(concurrency / load.ramp_up)
+            hatch = concurrency / load.ramp_up
```

Comment:
I would add `float()` around `load.ramp_up` to ensure float division. Because `concurrency` is always `int` and `ramp_up` theoretically can be `int`.

Reply:
@undera fixed in https://github.com/Blazemeter/taurus/pull/697/commits/5d5d859ecd1a5c87ac6779dc31d9727708f99842